src/garage/tf/plotter/plotter.py

```diff
@@ -53,11 +53,11 @@ def __init__(self,
                  rollout=default_rollout):
         Plotter.__plotters.append(self)
         self._env = env
-        self._policy = policy
         self.sess = tf.compat.v1.Session() if sess is None else sess
-
         self.graph = tf.compat.v1.get_default_graph(
         ) if graph is None else graph
+        with self.sess.as_default(), self.sess.graph.as_default():
```

Comment:
based on the line immediately above, shouldn't this be `self.graph.as_default()`?